src/Datadog.Trace/Logging/CustomLog4NetLogProvider.cs

```diff
@@ -82,10 +82,18 @@ protected override OpenMdc GetOpenMdcMethod()
         /// <returns>true if the log4net.Ext.Json assembly is not found or if it is found and the version is &gt;= 2.0.9.1. Otherwise, false.</returns>
         private static bool ExtJsonAssemblySupported()
         {
-            Assembly extJsonAssembly = Assembly.Load("log4net.Ext.Json");
-            if (extJsonAssembly is not null)
+            try
             {
-                return extJsonAssembly.GetName().Version >= SupportedExtJsonAssemblyVersion;
+                Assembly extJsonAssembly = Assembly.Load("log4net.Ext.Json");
+                if (extJsonAssembly is not null)
+                {
+                    return extJsonAssembly.GetName().Version >= SupportedExtJsonAssemblyVersion;
+                }
+            }
+            catch
```

Comment:
If the instrumented application doesn't use the JSON library then we'll always enter the catch block once during logs injection setup, but it doesn't necessarily mean that anything is wrong. So I preferred not writing any logs because that might cause confusion

Reply:
Maybe this could be a debug-level log?

Reply:
Added a log at the debug log level in a405f0d6c62581810788237a4e01168aacc4cb9a